src/Console/ModelsCommand.php

```diff
@@ -660,6 +660,10 @@ protected function createPhpDocs($class)
         $classname = $reflection->getShortName();
         $originalDoc = $reflection->getDocComment();
         $keyword = $this->getClassKeyword($reflection);
+        $interfaceNames = array_diff_key(
+            $reflection->getInterfaceNames(),
+            $reflection->getParentClass()->getInterfaceNames()
+        );
```

Comment:
What does this dark magic thing do? 🧙‍♂️ :)

Reply:
So `ReflectionClass::getInterfaceNames` get's the names of every interface that the underlying class satisfies. Since we only _need_ to  document this models interfaces (the parent interfaces automatically are implemented via inheritance), I find the diff to know what interfaces this model explicitly implements. We could go ahead and list out all of them, but it isn't necessary.  `getInterfaceNames` always returns an array, so this should be safe.

Reply:
Purrfect, thank you!